internal/workload/podspec_updates.go

```diff
@@ -475,20 +478,44 @@ func (s *updateState) updateContainer(p *cloudsqlapi.AuthProxyWorkload, wl Workl
 		params := map[string]string{}
 
 		// if it is a TCP socket
+		if inst.UnixSocketPath == "" {
+
+			port := s.useInstancePort(p, inst)
+			params["port"] = fmt.Sprint(port)
+			if inst.HostEnvName != "" {
```

Comment:
What do we do if HostEnvName and PortEnvName are empty strings?

Reply:
This is super helpful. Shall we document this somewhere?

Reply:
I added a new file `docs/authproxyworkload-reference.md` which contains the comment above. In future PRs, we can
add more configuration detail about how the AuthProxyWorkload resource should be used. This can be input for
our official documentation.